Pod/Classes/ios/NYTPhotosOverlayView.m

```diff
@@ -49,6 +50,10 @@ - (void)layoutSubviews {
     }];
     
     [super layoutSubviews];
+
+    if ([self.captionView respondsToSelector:@selector(setPreferredMaxLayoutWidth:)]) {
```

Comment:
That's really the best answer for "why check `respondsToSelector:` instead of `conformsToProtocol:`?" Just that the former requires the runtime lock while doing 60fps UI work.